websites/B/bilibili/presence.ts

```diff
@@ -3,7 +3,7 @@
  * @param {Number} videoTime Current video time seconds
  * @param {Number} videoDuration Video duration seconds
  */
-function getTimestamps(
+ function getTimestamps(
```

Comment:
Instead of this function use `presence.getTimestamps()` in line where args are passed and delete this function from code.

Reply:
ok i just performed this